docs/adr/0014-web-analytics.md

```diff
@@ -0,0 +1,28 @@
+# 14. Use New Relic to record and present web analytics
+
+Date: 2021-05-03
+
+## Status
+
+Proposed
+
+## Context
+
+We need to capture information about user behaviors and task completion on the website. We should use previously approved systems for capturing this information. This excludes services that capture and track web analytics in external systems, e.g. Google Analytics.
+
+## Decision
+
+We will use New Relic to capture and track web analytics. New Relic provides [browser monitoring](https://docs.newrelic.com/docs/browser/) that can captures essential metrics such as page views, and offers both [agent and SPA API](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-agent-spa-api/) and [APM](https://developer.newrelic.com/collect-data/custom-attributes) methods for capturing custom data.
+
+## Consequences
+
+We will install the New Relic Browser agent and enable SPA monitoring. This may require a change to our New Relic subscription and could entail additional costs.
```

Comment:
I don't know specifics about pricing, but using SPA requires selecting **Pro + SPA** so that indicates to me that a Pro account is required. I don't know what account type OHA currently has, and [New Relic's pricing page](https://newrelic.com/pricing) doesn't indicate what the costs for Pro are (unless you log in, I assume).